### ISSUE TITLE
DISABLED test_transformer_runtime (__main__.TestRuntimeEstimator)

### ISSUE THREAD

**Issue by jithunnair-amd** [pytorch/pytorch#134824]
Platforms: rocm

Failure introduced by https://github.com/pytorch/pytorch/pull/134243

This test was disabled because it is failing on main branch ([recent examples](https://torch-ci.com/failure?failureCaptures=%5B%22distributed%2F_tools%2Ftest_runtime_estimator.py%3A%3ATestRuntimeEstimator%3A%3Atest_transformer_runtime%22%5D)).

cc @jeffdaily @sunway513 @pruthvistony @ROCmSupport @dllehr-amd @jataylo @hongxiayang

**Comment by janeyx99** [comment]
cc @sanketpurandare 

@jithunnair-amd is there a timeline to re-enable these tests? If these are to be skipped more permanently, it would be better to migrate this to an in-code skip.

**Comment by jithunnair-amd** [comment]
Yes, we will be trying to resolve this within the next two weeks.

**Comment by kwen2501** [comment]
Cc @sanketpurandare @jithunnair-amd @janeyx99 @huydhn 

I am disabling the accuracy check on the time estimation because it breaks when we upgrade CI GPUs from M60 to T4.
